code/game/atoms.dm

```diff
@@ -14,21 +14,17 @@
 	var/germ_level = GERM_LEVEL_AMBIENT // The higher the germ level, the more germ on the atom.
 	var/simulated = TRUE //filter for actions - used by lighting overlays
 	var/atom_say_verb = "says"
-	var/dont_save = 0 // For atoms that are temporary by necessity - like lighting overlays
-
+	var/dont_save = FALSE // For atoms that are temporary by necessity - like lighting overlays
 
 	///Chemistry.
 	var/container_type = NONE
 	var/datum/reagents/reagents = null
 
 	//This atom's HUD (med/sec, etc) images. Associative list.
-	var/list/image/hud_list = list()
+	var/list/image/hud_list
```

Comment:
Be sure to test this thoroughly.

Reply:
Aye, atoms that use HUDs (mobs, hydroponics) call `prepare_huds` and HUDs indeed work in cases I checked (most of them)